src/main/api/publications/updateConfig.ts

```diff
@@ -11,6 +19,48 @@ const updateConfig: IpcEventHandler = async (
   const configHandler = createConfigFileHandler({ dirPath });
   const oldConfig = await configHandler.getConfig();
   await configHandler.setConfig({ ...oldConfig, ...changes });
+  await stageChanges(CONFIG_NAME, dirPath);
+
+  if (changes.name || changes.description) {
```

Comment:
@LostInStatic you can usually just return early with inverse condition instead of wrapping the code in an `if` in these situations.

Reply:
Sure, amended.